docs/application/dotnet/guides/tizen.circularUI/CircleSurfaceEffectBehavior.md

```diff
@@ -5,18 +5,23 @@ summary: CircleSurfaceEffectBehavior guide
 
 # CircleSurfaceEffectBehavior
 The `CircleSurfaceEffectBehavior` is an effect, which allows you to attach the `CircleSurface` to the [Xamarin.Forms.Page](https://developer.xamarin.com/api/type/Xamarin.Forms.Page/). [CircleSurface](https://developer.tizen.org/development/guides/native-application/user-interface/efl/ui-components/wearable-ui-components/circle-surface) is a Tizen specific component that manages [circle objects](https://developer.tizen.org/development/guides/native-application/user-interface/efl/ui-components/wearable-ui-components/circle-object).
-If you need to add `circle control`(such as [CircleListView](xref:Tizen.Wearable.CircularUI.doc.CircleListView), [CircleDateTimeSelector](xref:Tizen.Wearable.CircularUI.doc.CircleDateTimeSelector), [CircleScrollView](xref:Tizen.Wearable.CircularUI.doc.CircleScrollView), [CircleStepper](xref:Tizen.Wearable.CircularUI.doc.CircleStepper)) in the `Page` in Tizen Xamarin Forms application, there are two ways. One is inserting these controls to `CirclePage`, the other is adding `CircleSurfaceEffectBehavior` in the `Page` that includes `circle control`.
+
+Following are the two ways to add circle control such as [CircleListView](xref:Tizen.Wearable.CircularUI.doc.CircleListView), [CircleDateTimeSelector](xref:Tizen.Wearable.CircularUI.doc.CircleDateTimeSelector), [CircleScrollView](xref:Tizen.Wearable.CircularUI.doc.CircleScrollView) and [CircleStepper](xref:Tizen.Wearable.CircularUI.doc.CircleStepper) at `Page` in Tizen Xamarin Forms application:
+
+- Inserting these controls to `CirclePage`.
+- Adding `CircleSurfaceEffectBehavior` at `Page` that includes circle control.
 
 ## To add CircleSurfaceEffectBehavior in the Page
-Add `CircleSurfaceEffectBehavior` in the `<Page.Behaviors>`. And then set `RotaryFocusTargetName` property with `circle control` name. `RotaryFocusTargetName` sets the currently focused control that is handled by rotating and display the focused control's circle object. Please refer to below sample code.
 
-_The code example of this guide uses WearableUIGallery's TCListAppender code at the test\WearableUIGallery\WearableUIGallery\TC\TCListAppender.xaml_
+Add `CircleSurfaceEffectBehavior` in the `<Page.Behaviors>` and set `RotaryFocusTargetName` property with circle control name. `RotaryFocusTargetName` sets the currently focused control that is handled by rotating and display the focused control's circle object. Please refer to below sample code.
```

Comment:
Please get this sentence "Please refer to below sample code."  above the code ``xml
 <w:TwoButtonPage xmlns="http://xamarin.com/schemas/2014/forms"	 <w:TwoButtonPage xmlns="http://xamarin.com/schemas/2014/forms"...."

Please rephrase this sentence as:
The following is the sample code:
 
According to the checklist we don't use the word "Please" and "below" in the technical document.

Reply:
OK. I remove all "Please" and "below".